src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java

```diff
@@ -149,6 +158,46 @@ private String getOktaSessionToken() throws IOException {
         }
     }
 
+    private void multiProfileCreateIni() throws  IOException {
+
+        boolean result = false;
+        File okta = new File(System.getProperty("user.home") + "/.okta");
+        File profileIni = new File(System.getProperty("user.home") + "/.okta/.okta-profile-expiry");
```

Comment:
Path to the Okta dir should be written once and reused. 

```java
Path userHome = Paths.get(System.getProperty("user.home");
Path oktaDir = userHome.resolve(".okta");
Path profileIni = oktaDir.resolve(".okta-profile-expiry");
```

Since **oktaDir** is used outside this method, you might want to declare it as a field or declare a method that returns it.

Checking for existence:

```java
if (Files.exists(oktaDir)) {
```

Creating directory:

```java
Files.createDirectory(oktaDir);
```

Creating file:

```java
Files.createFile(profileIni);
```